dockers/docker-lldp/lldpmgrd

```diff
@@ -267,6 +269,15 @@ class LldpManager(daemon_base.DaemonBase):
         elif key == "PortConfigDone":
             self.port_config_done = True
 
+    def appdb_contains_lldp_port(self):
```

Comment:
let's make it utility function and add in device_info.py as rename something as `is_frontend_port_present_in_host()`

Reply:
> let's make it utility function and add in device_info.py as rename something as `is_frontend_port_present_in_host()`

Addressed the comment as the suggestion.  Define the utility function `is_frontend_port_present_in_host()' in the device_inof.py and use it in the lldpmgrd file